src/ios/Calendar.h

```diff
@@ -33,6 +33,6 @@
 - (void)listCalendars:(CDVInvokedUrlCommand*)command;
 
 - (void)deleteEvent:(CDVInvokedUrlCommand*)command;
-- (void)deleteEventFromNamedCalendar:(CDVInvokedUrlCommand*)command;
+- (void)deleteEventFromCalendar:(CDVInvokedUrlCommand*)command calendar: (EKCalendar *) calendar;
```

Comment:
The JS API calls into `deleteEventFromNamedCalendar`, but this no longer matches that. Can you please elaborate?